lib/plugins/aws/package/compile/events/lib/ensureApiGatewayCloudWatchRole.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
```

Comment:
Should we add a test for this file as well?

Reply:
Added

Reply:
Thanks :+1: